snooty/test_parser.py

```diff
@@ -1705,6 +1705,24 @@ def test_callable_target() -> None:
 </root>""",
     )
 
+    # Ensure that a missing argument doesn't crash
```

Comment:
So do we complain in the log if someone does
```
.. method:: 

   creates an index
```

Because I feel like we should… maybe that's handled elsewhere?

Reply:
We will with this change -- we check that the right diagnostic is raised a bit down!

Reply:
aaaah excellent